app/models/line_item.rb

```diff
@@ -0,0 +1,16 @@
+class LineItem < ApplicationRecord
+  belongs_to :deal
+  belongs_to :order
+
+  def unit_price
```

Comment:
let's have all these as columns so we can keep track of these prices in future even if tax field on deal is changed

Reply:
done